### ISSUE TITLE
set up list serves

### ISSUE THREAD

**Issue by mellybelly** [monarch-initiative/monarch-devops#17]
Let's set up the following lists:

info@monarchinitiative.org -> gets sent to @jmcmurry , cc all site PIs
monarch-dev -> all internal project members. Closed list.
community@monarchinitiative.org -> all internal project members plus additional members. list managed by @jmcmurry

**Comment by jnguyenx** [comment]
So we discussed a bit about mailing lists, and haven't chosen a solution
yet (OHSU mailman vs google groups).

I think that we can achieve what we want with GoDaddy email forwarding
service, but I don't see that very maintainable in the long run.
In my opinion the best would be to use GoDaddy email forwarding service and
redirect that to a proper mailing list.

Any thoughts Ethan?

**Comment by vanmatre** [comment]
I agree - Use GoDaddy email forwarding to a real list. Use google groups or ohsu mailman for the actual lists. I can set up ohsu mailman if you decide to use that mail robot.

**Comment by mellybelly** [comment]
@jnguyenx @vanmatre should we use google since OHSU mailman may have issues with external addresses? Is that true? I am happy to set up if that is our decision. Would like to get done this week. -thx

**Comment by vanmatre** [comment]
I think that OHSU mailman can use external addresses. Almost positive. I have no preference. I've setup mailman in the past but not google.

**Comment by jmcmurry** [comment]
Before we commit to OHSU mailman, let's first be sure that non-OHSU
addresses can be accommodated.
An easy way to check this would be to try to add my gmail address to the
ODG list serve as this is something I have been eager to do anyway.

Best,
Julie

On Tue, Feb 16, 2016 at 9:18 AM, vanmatre <notifications@github.com> wrote:

> I think that OHSU mailman can use external addresses. Almost positive. I
> have no preference. I've setup mailman in the past but not google.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/monarch-initiative/monarch-devops/issues/17#issuecomment-184781538>
> .
>



-- 
----------------------------------
mcmurry.julie@gmail.com +1 805 455 5877 skype: mcmurry.julie git: jmcmurry
----------------------------------

**Comment by mellybelly** [comment]
I am pretty sure we can’t do that, shannon?

On Feb 16, 2016, at 9:26 AM, Julie McMurry <notifications@github.com<mailto:notifications@github.com>> wrote:

Before we commit to OHSU mailman, let's first be sure that non-OHSU
addresses can be accommodated.
An easy way to check this would be to try to add my gmail address to the
ODG list serve as this is something I have been eager to do anyway.

Best,
Julie

On Tue, Feb 16, 2016 at 9:18 AM, vanmatre <notifications@github.com<mailto:notifications@github.com>> wrote:

> I think that OHSU mailman can use external addresses. Almost positive. I
> have no preference. I've setup mailman in the past but not google.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/monarch-initiative/monarch-devops/issues/17#issuecomment-184781538>
> .
>



--
----------------------------------
mcmurry.julie@gmail.com<mailto:mcmurry.julie@gmail.com> +1 805 455 5877 skype: mcmurry.julie git: jmcmurry
----------------------------------

—
Reply to this email directly or view it on GitHub<https://github.com/monarch-initiative/monarch-devops/issues/17#issuecomment-184786109>.

Dr. Melissa Haendel

Associate Professor
Ontology Development Group, OHSU Library
www.ohsu.edu/library/ontology<http://www.ohsu.edu/library/ontology>
Department of Medical Informatics and Clinical Epidemiology
Oregon Health & Science University
haendel@ohsu.edu<mailto:haendel@ohsu.edu>
skype: melissa.haendel
503-407-5970
Appointments: Shanez De Silva desilva@ohsu.edu<mailto:desilva@ohsu.edu>

**Comment by mellybelly** [comment]
You can if you request it through mailman.

On Feb 16, 2016, at 9:28 AM, Melissa Haendel <haendel@ohsu.edu<mailto:haendel@ohsu.edu>> wrote:

I am pretty sure we can’t do that, shannon?

On Feb 16, 2016, at 9:26 AM, Julie McMurry <notifications@github.com<mailto:notifications@github.com>> wrote:

Before we commit to OHSU mailman, let's first be sure that non-OHSU
addresses can be accommodated.
An easy way to check this would be to try to add my gmail address to the
ODG list serve as this is something I have been eager to do anyway.

Best,
Julie

On Tue, Feb 16, 2016 at 9:18 AM, vanmatre <notifications@github.com<mailto:notifications@github.com>> wrote:

> I think that OHSU mailman can use external addresses. Almost positive. I
> have no preference. I've setup mailman in the past but not google.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/monarch-initiative/monarch-devops/issues/17#issuecomment-184781538>
> .
>



--
----------------------------------
mcmurry.julie@gmail.com<mailto:mcmurry.julie@gmail.com> +1 805 455 5877 skype: mcmurry.julie git: jmcmurry
----------------------------------

—
Reply to this email directly or view it on GitHub<https://github.com/monarch-initiative/monarch-devops/issues/17#issuecomment-184786109>.

Dr. Melissa Haendel

Associate Professor
Ontology Development Group, OHSU Library
www.ohsu.edu/library/ontology<http://www.ohsu.edu/library/ontology>
Department of Medical Informatics and Clinical Epidemiology
Oregon Health & Science University
haendel@ohsu.edu<mailto:haendel@ohsu.edu>
skype: melissa.haendel
503-407-5970
Appointments: Shanez De Silva desilva@ohsu.edu<mailto:desilva@ohsu.edu>

**Comment by vanmatre** [comment]
CSLU has lists with external addresses in them on the OHSU mailman server. So we are confirmed.

--Ethna



Ethan VanMatre
Informatics Research Analyst
Institute on Development and Disability
Oregon Health & Science University
CSLU - GH40
3181 SW Sam Jackson Park Rd
Portland, OR 97239
(503) 346-3764
vanmatre@ohsu.edu
________________________________

**Comment by mellybelly** [comment]
great, go ahead and set up then please, I’m fine with the OHSU mailman as long as it meets our requirements.
Can we keep an archive of the mail? or does it go into the OHSU abyss?

On Feb 16, 2016, at 9:42 AM, vanmatre <notifications@github.com<mailto:notifications@github.com>> wrote:

CSLU has lists with external addresses in them on the OHSU mailman server. So we are confirmed.

--Ethna



Ethan VanMatre
Informatics Research Analyst
Institute on Development and Disability
Oregon Health & Science University
CSLU - GH40
3181 SW Sam Jackson Park Rd
Portland, OR 97239
(503) 346-3764
vanmatre@ohsu.edu<mailto:vanmatre@ohsu.edu>
________________________________

**Comment by vanmatre** [comment]
I'll start setting those lists up and check on the abyss....

--Ethan

Ethan VanMatre
Informatics Research Analyst
Institute on Development and Disability
Oregon Health & Science University
CSLU - GH40
3181 SW Sam Jackson Park Rd
Portland, OR 97239
(503) 346-3764
vanmatre@ohsu.edu
________________________________

**Comment by vanmatre** [comment]
I have created the 3 mailing lists and placed myself and Julie as list administrators. The forwarding from godaddy has be also set up. External address acceptance and delivery has been tested and is OK.

We now need a list of email addresses to go into each list. I can bulk load them (one per line please)

the lists (including alias and actual list) are:

info@monarchinitiative.org ====> monarch-info@ohsu.edu
monarch-dev@monarchinitiative.org ====> monarch-dev@ohsu.edu
community@monarchinitiative.org ====> monarch-community@ohsu.edu

the admin interface URL are:

http://mailman.ohsu.edu/cgi-bin/mailman/admin/monarch-info
http://mailman.ohsu.edu/cgi-bin/mailman/admin/monarch-dev
http://mailman.ohsu.edu/cgi-bin/mailman/admin/monarch-community

Regards, ethan

**Comment by jnguyenx** [comment]
@vanmatre it seems that @jmcmurry does not receive emails from info@monarchinitiative.org, nor @mellybelly.
Can you check that the address is valid to start with?

**Comment by vanmatre** [comment]
@mellybelly  is not in the info list ( I will add her). We do need a list of peoples email addresses to populate each list.

@jmcmurry is in the list but it is pointing to a gmail email address. Since Yahoo, Gmail and other started to use strict DMARC mailman (our listserve) has had problems with Yahoo etc rejecting mailman emails.

I will have to check with the OHSU mailman folks to find out what is happening.

In another list I manage I've givenup on mailman and moved to simplelists (a for money mailinglist service).


But back to the mailing lists - do we know who shouled be on each of the 3 lists?

**Comment by jnguyenx** [comment]
@vanmatre any updates on the address info@monarchinitiative.org? It seems that no one is currently checking it, which worries me.

Concerning the other lists, @jmcmurry should give you those info.

**Comment by vanmatre** [comment]
I mostly handle any administrative needs of the lists and let @jmcmurry handle any "substance".

Yahoo, etc has great problems with mailman and simplelists. I think that yahoo wants to rule all mailing lists.
